css/style.css

```diff
@@ -23,7 +23,7 @@ h2{font-family: 'Franklin Gothic Medium';
 div{
 display:block;
 }
-  .input-radio,
+  .input-radio,,
```

Comment:
Kindly create a new class for the "input-radio" or delete one of the commas to make it share the same attributes with the "input-checkbox".